Gemfile

```diff
@@ -39,6 +39,8 @@ gem 'bugsnag'
 # https://github.com/bugsnag/bugsnag-capistrano/blob/8bcfb27cf6eaff312eef086cce729d553a431460/UPGRADING.md
 gem 'bugsnag-capistrano', '< 2', require: false
 
+gem 'dotenv-rails', '~> 2.7'
```

Comment:
we already have it btw ... https://github.com/3scale/porta/blob/2a05a51a6ef35967f3c438ceff3a5d548f5bcd6a/Gemfile#L16

But I really want to remove it. I think it was used at the premise of the project ...

I do not believe an app should use a file to load the environments it needs ... This concept is already strange for the least...

I would understand if it is a launcher but not the app itself... 

For example unicorn zero-downtime restart used to not work well with those .env files
The author of dotenv really advises against using it in production anyway https://github.com/bkeepers/dotenv/issues/176#issuecomment-81792965
So at least use this only for development only. For test, I do not want to use it in a CI ... debugging can be hard.

So do you still want to use it in development?
I'd say you have other choices, like [direnv](https://github.com/direnv/direnv)  or [related projects](https://github.com/direnv/direnv#related-projects) 

Or just a (profile/zsh.bash.fish).rc like ... that you source before calling your executable ...
This is up to you to use, nothing coercing others and having hours of debugging session ending up with an env variable setup in a file not shown with `env` command ...